cleaners/winrar.xml

```diff
@@ -33,7 +35,7 @@
   <description>File archiver</description>
   <option id="private">
     <label>Private Data</label>
-    <description>Deletes the Private Data</description>
+    <description>Deletes the data</description>
```

Comment:
How about "Deletes the files"? It's already translated, and the label clarifies this is all about private data.

Reply:
But no files get deleted... AFAIK I used now "Delete the data" and this in a couple of cleaners... It's really universal - more universal then "Delete the files" (also plural) - so I think it's really worth (also for the future, for other cleaners), to translate this string...